README.md

```diff
@@ -27,7 +27,7 @@ make all
 ctest -V
 ```
 
-
+cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DBUILD_TESTS=ON  CMAKE_C_FLAGS_RELWITHDEBUBINFO="-03 -g -DNDEBUG" ..
```

Comment:
necessary with O3?

Reply:
The error appeared only in O3 in clang, so I'd say yes, for testing

Reply:
Ok, agreed.
I vouch for merge - Felix?